app/components/lists/navbar_links.js

```diff
@@ -2,47 +2,70 @@ const NAVBAR_LINKS = [
   {
     id: "navbar-1",
     section: "Sobre mim",
+
```

Comment:
please remove this line

also, why some objects do not have imgUrl or description?

finally, the images should be in the s3 bucket

Reply:
done! well noted, tahnks.
the imgUrl and description were included for all objects to keep the consistency 
the images were not uploaded in s3 bucket because they are test-images - the official images are not available yet